openedx_caliper_tracking/processor.py

```diff
@@ -62,7 +63,15 @@ def deliver_caliper_event(event):
                     settings.CALIPER_DELIVERY_AUTH_TOKEN),
                 'Content-Type': 'application/json',
             },
-            json=event)
+            json={
+                "records": [
+                    {
+                        "key": event_type,
+                        "value": caliperized_event
```

Comment:
While reading the documentation of kafka, I came across these lines
```
The Kafka cluster stores streams of records in categories called topics.
Each record consists of a key, a value, and a timestamp.
```

So I think we will also have to send `timestamp` within the record.